Mathlib/GroupTheory/GroupAction/Basic.lean

```diff
@@ -244,14 +244,44 @@ theorem stabilizer_smul_eq_stabilizer_map_conj (g : G) (a : α) :
   rw [mem_stabilizer_iff, ← smul_left_cancel_iff g⁻¹, smul_smul, smul_smul, smul_smul,
     inv_mul_cancel, one_smul, ← mem_stabilizer_iff, Subgroup.mem_map_equiv, MulAut.conj_symm_apply]
 
+/-- The natural group equivalence between the stabilizers of two elements in the same orbit. -/
+def stabilizerEquivStabilizer {a b : α} {g : G} (hg : g • b = a) :
+    stabilizer G b ≃* stabilizer G a :=
+  ((MulAut.conj g).subgroupMap <| stabilizer G b).trans
+    (MulEquiv.subgroupCongr (by
+      simp [← hg, stabilizer_smul_eq_stabilizer_map_conj]))
+
+theorem stabilizerEquivStabilizer_apply {a b : α} {g : G} (hg : g • b = a) (x : stabilizer G b) :
+    stabilizerEquivStabilizer hg x = MulAut.conj g x := by
+  -- simp [stabilizerEquivStabilizer]
+  simp only [stabilizerEquivStabilizer, MulEquiv.trans_apply, MulEquiv.subgroupCongr_apply,
+    MulEquiv.coe_subgroupMap_apply, MulAut.conj_apply]
+
+theorem stabilizerEquivStabilizer_symm_apply
+    {a b : α} {g : G} (hg : g • b = a) (x : stabilizer G a) :
+    (stabilizerEquivStabilizer hg).symm x = MulAut.conj g⁻¹ x := by
+  simp [stabilizerEquivStabilizer]
+
+theorem stabilizerEquivStabilizer_trans
+    {a b c : α} {g h k : G} {hg : g • b = a} {hh : h • c = b} {hk : k • c = a} (H : k = g * h) :
+    (stabilizerEquivStabilizer hh).trans (stabilizerEquivStabilizer hg)
+      = (stabilizerEquivStabilizer hk) := by
+  ext x; simp [stabilizerEquivStabilizer_apply, H]
+
+theorem stabilizerEquivStabilizer_one {a : α} :
+    stabilizerEquivStabilizer (one_smul G a) = MulEquiv.refl (stabilizer G a) := by
+  ext; simp [stabilizerEquivStabilizer_apply]
+
+theorem stabilizerEquivStabilizer_inv {a b : α} {g : G} (hg : g • b = a) :
+    let hg' : g⁻¹ • a = b := inv_smul_eq_iff.mpr (id (Eq.symm hg))
+    stabilizerEquivStabilizer hg' = (stabilizerEquivStabilizer hg).symm := by
```

Comment:
```suggestion
    stabilizerEquivStabilizer (inv_smul_eq_iff.2 hg.symm) = (stabilizerEquivStabilizer hg).symm := by
```

Reply:
(I haven't tested this, but I expect it should work!)

Reply:
It works, indeed.